### DISCUSSION TITLE
DNS rewrite TTL

### DISCUSSION THREAD

**Discussion by gmichels** [AdguardTeam/AdGuardHome#3942] [Q&A] [answered]
Hello,

I am successfully doing a DNS rewrite in the client filtering rules, such as:

    ||example.com^$dnsrewrite=NOERROR;A;192.168.0.100

However the rewritten result has a very short TTL (10 seconds). Is there any way to control the TTL of this rule?

Thank you

**Comment by ainar-g** [comment, marked as answer]
Hello.  It's controlled by the `blocked_response_ttl` setting in the configuration file.

**Reply by gmichels** [reply]
Exactly what I was looking for. Thanks!
